numpy/core/src/multiarray/mapping.c

```diff
@@ -2433,35 +2429,38 @@ mapiter_fill_info(PyArrayMapIterObject *mit, npy_index_info *indices,
 
     return 0;
 
-  broadcast_error:
+broadcast_error: ; // need semicolon to avoid compilation Cerror
```

Comment:
And I thought I knew the C language... Perhaps
```suggestion
broadcast_error: ; // `;` is an empty statement, goto labels cannot precede declarations
```

Reply:
Took me a while to track that down. The error message was clear, but so weird that it took me a while to comprehend and track down the solution.